src/Audio/Cue.cs

```diff
@@ -774,6 +783,14 @@ internal bool INTERNAL_update()
 								long elapsedFromStop = INTERNAL_timer.ElapsedMilliseconds - INTERNAL_fadeStart;
 								variableValue = elapsedFromStop;
 							}
+							else
+							{
+								variableValue = 0.0f;
+							}
+						}
+						else if (curRPC.Variable.Equals("NumCueInstances"))
```

Comment:
Doesn't the GetVariable() call before this if block handle NumCueInstances already?